### ISSUE TITLE
[Feature/Improvement] Run the timer in the background

### ISSUE THREAD

**Issue by a-mabe** [a-mabe/OpenHIIT#58]
**Purpose:** As a user, I want the timer to continue running even if OpenHIIT isn't the currently active app.

**More details:** The timer should continue running even if another app becomes active. For example, if the user opens their music app during a workout the timer should continue counting down.

**Acceptance Criteria:**
- When OpenHIIT is not the active app, the timer should continue counting down.
- Sound effects should also play when OpenHIIT is not the currently active app.

**Comment by a-mabe** [comment]
Work on the background timer is happening here: https://github.com/a-mabe/background_timer

The goal is to use the timer package as follows:

```
Countdown(
  controller: _controller,
  workSeconds: 8,
  restSeconds: 5,
  numberOfIntervals: 2,
  build: (_, int time) => Text(
    time.toString(),
    style: TextStyle(
      fontSize: 100,
    ),
  ),
  interval: Duration(milliseconds: 100),
  onFinished: () {

  },
),
```

**Comment by a-mabe** [comment]
```
V/BackgroundService( 1987): Service already running, using existing service
D/AndroidRuntime( 1987): Shutting down VM
E/AndroidRuntime( 1987): FATAL EXCEPTION: main
E/AndroidRuntime( 1987): Process: com.codepup.workout_timer, PID: 1987
E/AndroidRuntime( 1987): java.lang.NullPointerException: Attempt to invoke virtual method 'void io.flutter.plugin.common.MethodChannel.invokeMethod(java.lang.String, java.lang.Object)' on a null object reference
E/AndroidRuntime( 1987): 	at id.flutter.flutter_background_service.BackgroundService$2.run(BackgroundService.java:238)
E/AndroidRuntime( 1987): 	at android.os.Handler.handleCallback(Handler.java:942)
E/AndroidRuntime( 1987): 	at android.os.Handler.dispatchMessage(Handler.java:99)
E/AndroidRuntime( 1987): 	at android.os.Looper.loopOnce(Looper.java:201)
E/AndroidRuntime( 1987): 	at android.os.Looper.loop(Looper.java:288)
E/AndroidRuntime( 1987): 	at android.app.ActivityThread.main(ActivityThread.java:7918)
E/AndroidRuntime( 1987): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime( 1987): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
E/AndroidRuntime( 1987): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
```

**Comment by a-mabe** [comment]
Closed in #70
